### PR TITLE
Update Ember-CLI to 2.7.0

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: tests/acceptance/asset-tag-test.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 var chai = require('chai');
-var defaultOptions = require('../fixtures/default-options');
```
Comment (sir-dunxalot) [inline tests/acceptance/asset-tag-test.js]: I'm guessing all the removals in this file were jshint errors as the vars weren't getting used?

**Comment by sir-dunxalot** [comment]
This looks great 👍  As a collaborator you can merge, correct? If so, feel free to go ahead and merge this into master. Then I will pull down and release on NPM. I think I could also add you as a collaborator on the NPM package if that would help?

File: tests/acceptance/asset-tag-test.js (hunk 1 above)
Comment (rwwagner90) [inline tests/acceptance/asset-tag-test.js]: Bingo

**Comment by rwwagner90** [comment]
@sir-dunxalot yeah, I can merge. If you want to add me to npm, I can release as well.

**Pull request merged**